workspaces/tech-insights/plugins/tech-insights-maturity/README.md

```diff
@@ -0,0 +1,177 @@
+
+<img src="./src/img/gold.png" width="15" height="15" /> **Gold**: Components with this Rank have fulfilled all Bronze-, Silver-, and Gold-level Checks.
+
+## Installation
+
+### Install the plugin
+
+```bash
+# From your Backstage root directory
+yarn --cwd packages/app add @backstage-community/plugin-tech-insights-maturity
+```
+
+### Add Maturity Summary Card to the entity overview page:
+
+If you want to show Maturity Summary in the overview of any entity use `EntityMaturitySummaryCard`.
+Note: This applies to all types of entity.
+
+<img src="./img/maturitySummaryCard.png" width="700" height="350" />
+
+```tsx
+// packages/app/src/components/catalog/EntityPage.tsx
+
+import { EntityMaturitySummaryCard } from '@backstage-community/plugin-tech-insights-maturity';
+
+const overviewContent = (
+  <Grid container spacing={3} alignItems="stretch">
+    {entityWarningContent}
+    <Grid item md={3} xs={6}>
+      <EntityAboutCard variant="gridItem" />
+    </Grid>
+    <Grid item md={3} xs={6}>
+      <EntityCatalogGraphCard variant="gridItem" height={400} />
+    </Grid>
+    ...
+    <Grid item md={3} xs={6}>
+      <EntityMaturitySummaryCard />
+    </Grid>
+  </Grid>
+);
+```
+
+### Add Maturity Checks overview (Maturity Scorecards) page to the any Component EntityPage:
+
+Note: This only applies to entities of Kind: 'Component'
+
+<img src="./img/maturityScorecardContent.png" width="700" height="350" />
+
+```tsx
+// packages/app/src/components/catalog/EntityPage.tsx
+
+import { EntityMaturityScorecardContent } from '@backstage-community/plugin-tech-insight-maturity';
+
+const componentEntityPage = (
+  <EntityLayoutWrapper>
+    <EntityLayout.Route path="/" title="Overview">
+      {overviewContent}
+    </EntityLayout.Route>
+    <EntityLayout.Route path="/ci-cd" title="CI/CD">
+      {cicdContent}
+    </EntityLayout.Route>
+    ...
+    <EntityLayout.Route path="/maturity" title="Maturity">
+      <EntityMaturityScorecardContent />
+    </EntityLayout.Route>
+    ...
+  </EntityLayoutWrapper>
+);
+```
+
+### Add Maturity Summary overview page to System/Domain/Group:
+
+Note: This only applies to entities of Kind: 'System', 'Domain', or 'Group'
+
+<img src="./img/maturitySummaryCardContent.png" width="700" height="350" />
+
+```tsx
+// packages/app/src/components/catalog/EntityPage.tsx
+
+import { EntityMaturitySummaryCardContent } from '@backstage-community/plugin-tech-insight-maturity';
+
+const systemEntityPage = (
+  <EntityLayoutWrapper>
+    <EntityLayout.Route path="/" title="Overview">
+      {overviewContent}
+    </EntityLayout.Route>
+    ...
+    <EntityLayout.Route path="/maturity" title="Maturity">
+      <EntityMaturitySummaryCardContent />
+    </EntityLayout.Route>
+    ...
+  </EntityLayoutWrapper>
+);
+```
+
+### Add Maturity page:
+
+<img src="./img/maturityPage.png" width="700" height="350" />
+
+First make the Scorecard page available as route
```

Comment:
The term `Scorecard` resonates more with me, but I think this should be consistent with the name of the plugins. 

```suggestion
First make the Maturity page available as route
```